modules/transactions.js

```diff
@@ -46,7 +46,7 @@ function Transactions (cb, scope) {
 			transaction: scope.logic.transaction,
 		},
 	};
-	genesisblock = library.genesisblock;
+	genesisblock = scope.genesisblock;
```

Comment:
We don't use it as a part of library in this file, we access `genesisblock` closure variable. Do we want to remove `genesisblock` variable and use `library.genesisblock` instead?